inlang/source-code/sdk/src/generateProjectId.ts

```diff
@@ -5,10 +5,11 @@ export async function generateProjectId(repo: Repository, projectPath: string) {
 	if (!repo || !projectPath) {
 		return undefined
 	}
-	const repoMeta = await repo.getMeta()
+	const firstCommitHash = await repo.getFirstCommitHash()
+	const originHash = await repo.getOrigin({ safeHashOnly: true })
```

Comment:
do we need the origin hash? 

while safe has only should be "safe". do we need to take the risk? Just `firstCommitHash+projectPath` seems sufficient, no?

Reply:
If we don't need it, we can remove the `safeHashOnly` logic in lix too (reduce maintenance)

Reply:
@samuelstroschein 

@martin-lysk voted for using this as a salt in the project id to prevent being able to check if a repo specific repo was used by someone, i think its a good idea